### PR TITLE
r-gprofiler2

### PR DESCRIPTION
**Pull request by liiskolb**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by liiskolb** [comment]
@conda-forge/staged-recipes I'm getting errors like: "conda.exceptions.UnsatisfiableError: The following specifications were found to be incompatible with each other:"
Do you have any experience with this? What could be the solution for this?

**Comment by dbast** [comment]
@liiskolb Please update this PR by creating the recipe 100% via https://github.com/bgruening/conda_r_skeleton_helper ...

**Comment by liiskolb** [comment]
@dbast Thanks, I have tried that before :) 
I used it again and it still fails randomly on platforms. I think that my issue is related to the discussion in here: https://github.com/conda-forge/staged-recipes/pull/9546 and the bug report in here https://github.com/conda/conda-build/issues/3735

File: recipes/r-gprofiler2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    identifier conversion and mapping orthologous genes across species via ''g:Profiler''
+    (<https://biit.cs.ut.ee/gprofiler>).  The main tools are:  (1) ''g:GOSt'' - functional
+    enrichment analysis and visualization of gene lists;  (2) ''g:Convert'' - gene/protein/transcript
+    identifier conversion across various namespaces; (3) ''g:Orth'' - orthology search
+    across species; (4) ''g:SNPense'' - mapping SNP rs identifiers to chromosome positions,
+    genes and variant effects This package is an R interface corresponding to the 2019
+    update of ''g:Profiler'' and provides access to ''g:Profiler'' for versions ''e94_eg41_p11''
+    and higher. See the package ''gProfileR'' for accessing older versions from the
+    ''g:Profiler'' toolset.  '
+  license_family: GPL3
```
Comment (dbast) [inline recipes/r-gprofiler2/meta.yaml]:
```suggestion
  license_family: GPL2
```

File: recipes/r-gprofiler2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  license_file: LICENSE
```
Comment (dbast) [inline recipes/r-gprofiler2/meta.yaml]:
```suggestion
  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```

File: recipes/r-gprofiler2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    - r-viridislite
+
+test:
+  commands:
+    - $R -e "library('gprofiler2')"           # [not win]
+    - "\"%R%\" -e \"library('gprofiler2')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=gprofiler2
+  license: GPL (>= 2)
```
Comment (dbast) [inline recipes/r-gprofiler2/meta.yaml]:
```suggestion
  license: GPL-2
```

File: recipes/r-gprofiler2/LICENSE [hunk 4]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (dbast) [inline recipes/r-gprofiler2/LICENSE]: Please remove this file.

**Comment by dbast** [comment]
@liiskolb Seems like you had an older conda-build version installed while running the conda_r_skeleton_helper... See the update suggestions. See recipe is fine after that ... lets then hope those CI issues get resolved soon.

**Comment by liiskolb** [comment]
@dbast Thanks, I did the suggested changes. However, my conda-build version is 3.18.9. Is this old? Anyway, I will wait until the CI issues get resolved. Thanks again :)

**Comment by dbast** [comment]
3.18.9 should be fine and generate the recipe as it is now. 

The linux build is now successful, which is normally enough for a noarch recipe. The osx failure is still strange and it looks either like there are general CI/conda-build issues or some transient dependencies are not available or broken.

**Comment by dbast** [comment]
hah. only appveyor still fails.

**Comment by dbast** [comment]
@liiskolb Thanks for this contribution and welcome at conda-forge.

**Pull request merged**